src/mono/netcore/gen-xunit-runner/Program.cs

```diff
@@ -284,7 +284,7 @@ static int Main (string[] args)
 
 		// Despite a lot of effort, couldn't get dotnet to load these assemblies from the sdk dir, so copy them to our binary dir
 //		File.Copy ($"{sdkdir}/Microsoft.DotNet.PlatformAbstractions.dll", AppContext.BaseDirectory, true);
-		File.Copy ($"{sdkdir}/CoreFx.Private.TestUtilities.dll", AppContext.BaseDirectory, true);
+		File.Copy ($"{sdkdir}/TestUtilities.dll", AppContext.BaseDirectory, true);
```

Comment:
Curious, what is this program for?

Reply:
@stephentoub

Reply:
It's a tool that converts xunit tests into a simple app that doesn't depend on the xunit runner, we use it for bringing up webassembly on netcore mono as a lot of the xunit machinery doesn't work there.

Reply:
I see. Didn't know if this was for pre-consolidation.